deploy/kwok/env.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -e
+cd "$(dirname "$0")"
+
+: ${TMP_DIR:="/tmp"}
+cat <<EOF
+export KUBECONFIG="${KUBECONFIG:-"$TMP_DIR/kubeconfig.yaml"}"
+
+export NUODB_CP_URL_BASE="http://localhost:8080"
+export NUODB_CP_USER="${NUODB_CP_USER:-"system/admin"}"
+export NUODB_CP_PASSWORD="${NUODB_CP_PASSWORD:-"changeIt"}"
+
+export PAUSE_OPERATOR_COMMAND="$(pwd)/pause-operator.sh"
+export RESUME_OPERATOR_COMMAND="$(pwd)/resume-operator.sh"
+
+export POD_SCHEDULING_ENABLED="false"
```

Comment:
Kwok does not schedule pods by default?
From what I've read it seems that it can be configured to mimic real pod lifecycle and simulate databases becoming ready. It might not be practical for negative testing though.

Reply:
Right. This is to differentiate Minikube and other real K8s clusters that actually schedule containers. I'll change it to `CONTAINER_SCHEDULING_ENABLED` to make it clearer.